src/twisted/web/http.py

```diff
@@ -319,11 +323,26 @@ class _MultiPartParseException(Exception):
     """
 
 
-def _getMultiPartArgs(content: bytes, ctype: bytes) -> dict[bytes, list[bytes]]:
+@dataclass
+class FileUpload:
+    """
+    A file that was uploaded to the web server as part of multipart request.
+    """
+
+    data: BinaryIO
+    name: str
+    filename: str
```

Comment:
Docstrings for these attributes would be good, because as someone without deep knowledge of HTTP this pair of attributes is going to look nonsensical.